wiki/Beatmap_Editor/Timing/en.md

```diff
@@ -1,41 +1,21 @@
-
-How to time songs
------------------
+_For a guide on timing, see: [How to time songs](/wiki/How_to_time_songs)_
 
-***Main page: [How to time songs](/wiki/Beatmap_Editor_Guides/How_to_time_songs)***
-
-Timing is extremely important. To time the song, you need to switch to timing tab.
-At the timing tab, there is a small box at the upper-right section, beside the timeline. This box will be used to set the approximate BPM based on your clicks.
-
-The mapper will play the song and listen to it. Press **«Tap Here»** in rhythm of the music.
-After a certain number of press, the approximate BPM will be set and the black keys will resonate as «tic toc toc toc» where tic is the start of new time signature (by 4/4 standard). Repeat at least five times and take the average reading. If you are not sure, ask for help in Chat Console or the forum. Upload your beatmap and use the link for request. You may have to do something in return.
-
-Again after the button is pressed enough, BPM sensitivity will change from 0,1 and 2.
+![Timing menu](img/Timing_base.jpg "Timing menu")
 
--   BPM Sensitivity 0: 999,00
--   BPM Sensitivity 1: 999,90
--   BPM Sensitivity 2: 999,99
-    -   *9 = random number / 0 = unaffected number.*
+The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and BPM (beats per minute). The song's offset is the amount of time between the start of an audio file and the first downbeat in the song. The song's BPM is essentially how fast a song is.
 
-**The button «T» may also be used instead of «Tap Here!»**
+Correct timing is imperative in the beatmapping process to allow placing objects in time with the beat.
 
-Timing Setup
-------------
+## Setup
 
-![The Timing Setup dialog showing a map with a fluctuating BPM and sampleset changes.](img/TimingSetup.jpg "The Timing Setup dialog showing a map with a fluctuating BPM and sampleset changes.")
+![Screenshot of the "Timing and Control Points" window](img/TimingSetup.jpg "The timing setup dialog shows the beatmap's fluctuating BPM and sampleset changes")
 
-Timing sections allow you to vary the timing across the song. There are two types of timing sections, a regular timing section and an inherited timing section.
+Timing sections allow you to vary the timing across the song. There are two types of timing sections: a regular timing section and an inherited timing section.
 
-A regular timing section is displayed as a red line in the editor. Every map has at least one. Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
+A regular timing section is displayed as a red line in the editor. Every beatmap must have at least one. Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
```

Comment:
However, some beatmaps change their BPM during the song and require multiple regular timing sections as a result.
don't use the `map` word i suppose

Reply:
😕 the entire article doesn't use `map` anywhere

Reply:
`Some maps change their BPM`